### PR TITLE
Build localstack-ext recipe

### PR DESCRIPTION
**Pull request by mwilson8**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mwilson8** [comment]
@conda-forge/help-python is it possible to add noarch:python even though this package uses java as well?

File: recipes/localstack-ext/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: bb7d3953e45af2df720b1327ce0c12d20b88fd03a63b0291ca9a132c748eb628
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (marcelotrevisani) [inline recipes/localstack-ext/meta.yaml]: Could it be ``noarch``?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

Comment (marcelotrevisani) [reply recipes/localstack-ext/meta.yaml]: Please see
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

**Comment by mwilson8** [comment]
@conda-forge/help-python 
@marcelotrevisani 
This package is ready for final review & merge

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**